### ISSUE TITLE
Noticible audio delay compared to video when using airplay audio output

### ISSUE THREAD

**Issue by jmesmon** [jellyfin/Swiftfin#420]
### Describe the bug

1. Set the audio output for the apple tv to airplay speakers (I do this by changing the default output in the apple tv settings, and am using a pair of homepods in a stereo group)
2. Play a video from swiftfin
3. note that the audio is somewhat delayed compared to the video. After pressing back or otherwise pausing, the audio continues playing for a few seconds. Other video streaming apps (Netflix, for example) have video and audio syncronized in this same configuration, and respond immediately to pausing the audio (they don't continue playing it for a few seconds)

### Application version

1.0.0(74)

### Where did you install the app from?

TestFlight

### Device information

Apple TV 4K, 32GB, A2169

### OS version

tvOS 15.4.1 (19L452)

### Jellyfin server version

10.7.7

**Comment by LePips** [comment]
Thank you for the issue, however this is an issue of VLCKit and there is nothing that we can do about it.

**Comment by jmesmon** [comment]
For reference, this appears to also affect bluetooth (though to a lesser degree because the bluetooth latency is lower).

Is there an open issue for this in VLCKit? If it is fixed in VLCKit, would any changes need to be made to Swiftfin to resolve this, or is this completely handled within VLCKit?

**Comment by LePips** [comment]
Over the few years it must have come back up, with a more recent issue:

https://code.videolan.org/videolan/VLCKit/-/issues/544

**Comment by Sapd** [comment]
It is an issue for me, too. Makes it completely unusable.
 I guess a lot of people use the Apple TV together with Homepods, probably affecting not few users.

**Comment by Sapd** [comment]
Seems to be fixed with the latest release. Tested it with multiple movies.

**Comment by edeustace** [comment]
Note that the native player does not have this issue, so you can switch to that if you're stuck.

**Comment by JPKribs** [comment]
There are a few duplicates of this issue that have been made but please find my response here (on one of the closed issues 🤦‍♂️)

https://github.com/jellyfin/Swiftfin/issues/734#issuecomment-2723290284

The TL;DR is this issue is related to VLCKit and the issue is tracked on their repo. If there is any movement/update to VLCKit we will update it to incorporate the fix. That URL is correct. That issue on the VLCKit side is a bit broad but specifically references "Delays".

On our end, I've left #937 open to track this as to try and consolidate communication around this. That being said, it's still a bit more scattered than I'd like but that's unfortunately the nature of the beast wrangling a project like this.

In summary, we do not have a route forward to solve this unless VLC resolves this issue on their end or we replace VLCKit on our end. The latter is a tall ask and would put much of our other work on pause. We have less than a handful of devs consistently contributing to Swiftfin so we appreciate everyone's patience as we work to improve Swiftfin as quickly as we can! 

If anyone is interested in helping out, please find our help wanted post here:

https://github.com/jellyfin/Swiftfin/discussions/1503
